src/gui/optionsdlg.cpp

```diff
@@ -974,19 +977,32 @@ void OptionsDialog::loadOptions()
     m_ui->spinMaxActiveTorrents->setValue(session->maxActiveTorrents());
     m_ui->checkIgnoreSlowTorrentsForQueueing->setChecked(session->ignoreSlowTorrentsForQueueing());
 
-    if (session->globalMaxRatio() >= 0.) {
+    float globalMaxRatio = session->globalMaxRatio();
```

Comment:
Negligible part of people use `-flto` with GCC and as such getter has to be defined in the header class to be inlined. But this is prohibited by the coding guidelines in this project.